Core/Base/Utils/FileManager.php

```diff
@@ -0,0 +1,349 @@
+<?php
+/**
+ * This file is part of FacturaScripts
+ * Copyright (C) 2018 Carlos Garcia Gomez <carlos@facturascripts.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * long with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace FacturaScripts\Core\Base\Utils;
```

Comment:
Don't use Utils as a namespace. Don't create new namespaces in base. Move this to Lib

Reply:
If we move to Lib this can be overwritted in a Plugin, and this integrity test can be completly nulled, resulting in false check.

Exist another way to prevent this according to this changes?